### PR TITLE
Added support for modern platforms

### PR DESCRIPTION
**Pull request by SomeAnon42**
.NET ecosystem is growing. It used to be just .NET Framework and Mono, but now there's .NET Core, Xamarin and UWP. I think it's a good time to target [.NETStandard](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md) to support them all.

### CODE REVIEW

**Comment by ioctlLR** [comment]
Can you adjust the pull request so it doesn't have file changes that are unnecessary to the updated build chain?  I see formatting changes that are spurious at best...

**Comment by SomeAnon42** [comment]
Not sure what *formatting* changes you're talking about, but yeah, there's some unnecessary changes that I can undo.

A separate project sounds good.

**Comment by SomeAnon42** [comment]
Better just wait until VS 2017 comes out. There'll be no more need to modify any code to consume libraries built for outdated versions of .NET (such as NVorbis) from modern apps (.NET 4.6+, UWP, .NET Core, Xamarin). I'm unhappy with the current state of the .NET Standard tooling in VS 2015 anyway, so let's not deal with .NET Standard just yet.

TL;DR: with the release of VS 2017 the problem with consuming your library from modern .NET projects will just go away,

I assume I'll delete the PR once you reply.

**Comment by ioctlLR** [comment]
Works For Me!  Thanks for the info.
